content/sales.rst

```diff
@@ -0,0 +1,152 @@
+Sales and Marketing
+###################
+
+:menu_order: 008
+
+.. contents::
+
+
+Introduction
+============
+
+This page briefly describes operational Sales details.
+
+
+Product Demo
+============
+
+Please see the `DemoSession` wiki page for the demo session structure.
```

Comment:
I think that this should be remove, we should not make refernces to things with don't exists. and for the purpose of GECK the private wiki don't exist